test/end-to-end/shared/components/bhFiscalYearSelect.js

```diff
@@ -0,0 +1,19 @@
+/* global browser, element, by */
+
+const FU = require('../FormUtils');
+
+module.exports = {
+  mainSelector : '[bh-fiscal-year-select]',
+  set : function set(fiscal_year_id, id) {
+      var opts;
+    /** return selector for an option element */
+    function getValue(id) {
```

Comment:
Just FYI - you can use `FormUtils` to make this all one line.  The method you are looking for is [select()](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/test/end-to-end/shared/FormUtils.js#L82).

```js
set: function (fiscalYearText, id) {
  const locator = id ? by.id(id) : by.css(this.mainSelector);
  FU.select('$ctrl.selectedFiscal', fiscalYearText, element(locator));
};
```